charts/cloudzero-agent/templates/cm.yaml

```diff
@@ -33,6 +33,9 @@ data:
         enable_compression: true
         follow_redirects: true
         enable_http2: true
+        static_configs:
+          - targets:
+            - {{ printf "%s.%s.svc.cluster.local:%d" .Values.kube_state_metrics.fullnameOverride .Release.Namespace (int .Values.kube_state_metrics.service.port) }}
```

Comment:
This resolves to something like: `cloudzero-state-metrics.cz-prom-agent.svc.cluster.local:8080`